Terminal.Gui/ConsoleDrivers/CursesDriver/CursesDriver.cs

```diff
@@ -689,6 +695,17 @@ public override void Init (Action terminalResized)
 			} catch (Exception e) {
 				Console.WriteLine ("Curses failed to initialize, the exception is: " + e);
 			}
+
+			// 
+			// We are setting Invisible as default so we could ignore XTerm DECSUSR setting
+			//
+			switch (Curses.curs_set (0)) {
+				case 0:		currentCursorVisibility = initialCursorVisibility = CursorVisibility.Invisible;	break;
+				case 1:		currentCursorVisibility = initialCursorVisibility = CursorVisibility.Underline;	Curses.curs_set (1); break;
+				case 2:		currentCursorVisibility = initialCursorVisibility = CursorVisibility.Box;		Curses.curs_set (2); break;
+				default:	currentCursorVisibility = initialCursorVisibility = null;						break;
+			}
```

Comment:
Let us follow the coding style, statements should be in different lines, and we should remove the additional space